### PR TITLE
Convai wild evaluation dataset task

### PR DESCRIPTION
**Pull request by litinsky**
Dataset collected during the wild evaluation of ConvaAI2 participants bots (http://lnsigo.mipt.ru/export/datasets/convai/convai2_wild_evaluation_0.2.tgz). 1921 convaersations. 60% train, 20% valid and 20% test is chosen at random from the whole dataset.

### CODE REVIEW

File: parlai/tasks/task_list.py [hunk 1]
```diff
@@ -411,4 +411,11 @@
         "tags": ["All", "Visual"],
         "description": "COCO annotations derived from the 2015 COCO Caption Competition. Link to dataset: http://cocodataset.org/#download",
     },
+    {
+        "id": "ConvAI2_wild_evaluation",
+        "display_name": "ConvAI2_wild_evaluation",
+        "task": "convai2_wild_evaluation",
+        "tags": ["All",  "ChitChat" ],
+        "description": "Dataset collected during the wild evaluation of ConvaAI2 participants bots (http://lnsigo.mipt.ru/export/datasets/convai/convai2_wild_evaluation_0.2.tgz). 60% train, 20% valid and 20% test is chosen at random from the whole dataset."
```
Comment (jaseweston) [inline parlai/tasks/task_list.py]: best not to include the tgz here as it will likely be updated, but could include http://convai.io perhaps?

Comment (litinsky) [reply parlai/tasks/task_list.py]: jaseweston, shure, fixed it.

File: parlai/tasks/convai2_wild_evaluation/build.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+def make_parlai_format(data: list, dpath: str):
+    train_p = 0.6
+    valid_p = 0.2
+    test_p = 1 - (train_p + valid_p)
+
+    assert train_p > 0
+    assert valid_p > 0
+    assert test_p > 0
+
+    random.seed()
```
Comment (jaseweston) [inline parlai/tasks/convai2_wild_evaluation/build.py]: we don't want a random seed changing the dataset for everyone who downloads it here

Comment (jaseweston) [reply parlai/tasks/convai2_wild_evaluation/build.py]: probably better to just take every other n lines or so?

**Comment by stephenroller** [comment]
Does this fix #1056?

File: parlai/tasks/convai2_wild_evaluation/build.py (hunk 2 above)
Comment (litinsky) [reply parlai/tasks/convai2_wild_evaluation/build.py]: jaseweston, sorry for being absent - was on my vacation. Removed random seed from the dataset extraction.

**Pull request merged**